hooks/nvidia-bootstrap/Makefile

```diff
@@ -16,8 +16,10 @@ all: image
 
 .PHONY: image push
 
-image:
-	docker build -t kopeio/nvidia-bootstrap:1.6.0 -f image/Dockerfile image/
+image:  ## Build the images
+    docker build --build-arg CUDA_VERSION=9.1 -t qntfy/nvidia-device-plugin:0.2.0-CUDA-9.1 -f image/Dockerfile image/
```

Comment:
I assume that before we merge this we'll need to switch this back to `kopeio` and have the image built and pushed by the people with permissions to that repo, correct?